### ISSUE TITLE
Double events are fired for Philips Wall Modules & Philips Hue Dimmer Switch

### ISSUE THREAD

**Issue by inlet** [Koenkk/zigbee-herdsman-converters#7323]
After updating the Zigbee2Mqtt add-on in home assistant yesterday suddenly events for my Philips Hue Wall modules and Hue dimmer Switch are fired twice.

After watching the logs I noticed that it indeed gets fired twice in a row at the same time.

Any help is highly appreciated! Thanks 🙏

When I flip the Philip Hue Wall module, this will be fired by Zigbee2Mqtt (logs):

```diff
Zigbee2MQTT:info  2024-04-03 13:59:52: MQTT publish: topic 'zigbee2mqtt/Kantoor Patrick Schakelaar', payload '{"action":"left_press","battery":100,"device_mode":"single_rocker","last_seen":"2024-04-03T13:59:52+02:00","linkquality":32}'

Zigbee2MQTT:info  2024-04-03 13:59:52: MQTT publish: topic 'zigbee2mqtt/Kantoor Patrick Schakelaar', payload '{"action":"","battery":100,"device_mode":"single_rocker","last_seen":"2024-04-03T13:59:52+02:00","linkquality":32}'

+ Zigbee2MQTT:info  2024-04-03 13:59:52: MQTT publish: topic 'zigbee2mqtt/Kantoor Patrick Schakelaar/action', payload 'left_press'

Zigbee2MQTT:info  2024-04-03 13:59:52: MQTT publish: topic 'zigbee2mqtt/Kantoor Patrick Schakelaar', payload '{"action":"left_press","battery":100,"device_mode":"single_rocker","last_seen":"2024-04-03T13:59:52+02:00","linkquality":36}'

Zigbee2MQTT:info  2024-04-03 13:59:52: MQTT publish: topic 'zigbee2mqtt/Kantoor Patrick Schakelaar', payload '{"action":"","battery":100,"device_mode":"single_rocker","last_seen":"2024-04-03T13:59:52+02:00","linkquality":36}'

+ Zigbee2MQTT:info  2024-04-03 13:59:52: MQTT publish: topic 'zigbee2mqtt/Kantoor Patrick Schakelaar/action', payload 'left_press'

Zigbee2MQTT:info  2024-04-03 13:59:52: MQTT publish: topic 'zigbee2mqtt/Kantoor Patrick Schakelaar', payload '{"action":"left_press_release","battery":100,"device_mode":"single_rocker","last_seen":"2024-04-03T13:59:52+02:00","linkquality":32}'

Zigbee2MQTT:info  2024-04-03 13:59:52: MQTT publish: topic 'zigbee2mqtt/Kantoor Patrick Schakelaar', payload '{"action":"","battery":100,"device_mode":"single_rocker","last_seen":"2024-04-03T13:59:52+02:00","linkquality":32}'

+ Zigbee2MQTT:info  2024-04-03 13:59:52: MQTT publish: topic 'zigbee2mqtt/Kantoor Patrick Schakelaar/action', payload 'left_press_release'

Zigbee2MQTT:info  2024-04-03 13:59:52: MQTT publish: topic 'zigbee2mqtt/Kantoor Patrick Schakelaar', payload '{"action":"left_press_release","battery":100,"device_mode":"single_rocker","last_seen":"2024-04-03T13:59:52+02:00","linkquality":32}'

Zigbee2MQTT:info  2024-04-03 13:59:52: MQTT publish: topic 'zigbee2mqtt/Kantoor Patrick Schakelaar', payload '{"action":"","battery":100,"device_mode":"single_rocker","last_seen":"2024-04-03T13:59:52+02:00","linkquality":32}'

+ Zigbee2MQTT:info  2024-04-03 13:59:52: MQTT publish: topic 'zigbee2mqtt/Kantoor Patrick Schakelaar/action', payload 'left_press_release'
```

> These double events only happen to these devices, other devices are triggered just fine

**Comment by inlet** [comment]
![image](https://github.com/Koenkk/zigbee-herdsman-converters/assets/232559/7468f6c0-abde-474c-903a-cab0e0764eb5)

**Comment by inlet** [comment]
This seems to be related https://github.com/Koenkk/zigbee2mqtt/issues/22073

**Comment by hvorragend** [comment]
I have the same problem and have already mentioned it here [#22073](https://github.com/Koenkk/zigbee2mqtt/issues/22073).

**Comment by inlet** [comment]
As a temporary workaround I've added a `debounce: 0.1` for the particular devices. However this give issues if you trigger the device without the interval rate

**Comment by inlet** [comment]
@Koenkk thanks so much 🙏! Yes this does solve the issue

**Comment by inlet** [comment]
Oops I was too quick with closing this issue.. removing all bindings and reconfigure the device doesn’t seem to help. Neither re-adding the device. Any help is appreciated, thanks in advance 🙏

**Comment by inlet** [comment]
**Update:**

The bind page (frontend -> device -> bind), contains 2 bindings. If I remove the last one everything works correctly. However, when I reconfigure it (yellow refresh icon), there are 2 bindings again and the events are triggered double again.

For now I just removed the last binding so there's only 1 binding with the coordinator. Hopefully this gives you a bit of contexts. Not sure what causes it, but it occurred after the latest z2m update. I tried to restore to a previous backup, but somehow the changes have been propagated to the zigbee network.
